### DISCUSSION TITLE
DeepFaceLab

### DISCUSSION THREAD

**Discussion by blaze8886** [community/community#22684] [New to GitHub]
<p>How and where can I download DeepFaceLab? I can’t find the link.:unamused:</p>

**Comment by MChevy422** [comment]
<p>Hello <a class="mention" href="/blaze8886">@blaze8886</a>, </p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and Github board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Greatly appreciated,</p>
<p><strong>-MChevy422</strong></p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/blaze8886">@blaze8886</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>You can simply clone the repository that you can find here: <a href="https://github.com/iperov/DeepFaceLab" rel="nofollow noopener">iperov/DeepFaceLab</a> and then follow the build guide line here: <a href="https://github.com/iperov/DeepFaceLab/blob/master/doc/doc_build_and_repository_info.md" rel="nofollow noopener">doc_build_and_repository_info</a>.</p>
<p>Ask if you have some problem</p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with<span> </span>Accept as Solution<span> </span>to help other users locate important info. Don’t forget to give<span> </span>Kudos<span> </span>for great contents!</p>

**Comment by blaze8886** [comment]
<p>Thank you! Is there a way to run deepface with 32bit and cpu or is there an other software?</p>

**Comment by jcn815** [comment]
<p>Don’t run 32bit… solution plzzz</p>
